pip_audit/_format/cyclonedx.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, result: Dict[service.Dependency, List[service.VulnerabilityRe
         super().__init__()
 
         for (dep, vulns) in result.items():
+            # TODO(alex): Is there anything interesting we can do with skipped dependencies in
```

Comment:
From looking at CycloneDX, I don't think there's a concept which captures this, so skipping is probably the right move.